Rubberduck.Core/UI/CodeExplorer/Commands/ImportCommand.cs

```diff
@@ -204,6 +522,37 @@ protected override void OnExecute(object parameter)
             }
         }
 
-        protected IDictionary<string, ComponentType> ComponentTypeForExtension { get; }
+
+        //We only allow extensions to be imported for which we might be able to determine that the conditions are met to actually import the file.
+        //The exception are specif exceptions to the rule.
```

Comment:
that line is a bit redundant, maybe reformulating around "There are specific exceptions to this rule stored in AlwaysImportableExtensions"